sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/webkey/CertificateKeyCurveName.java

```diff
@@ -9,7 +9,7 @@
 /**
  * Defines values for KeyCurveName.
```

Comment:
Defines values for Certificate Key Curve Name.
here and below.

Reply:
fixed.